src/templates/button-back.jsx

```diff
@@ -25,14 +25,14 @@ export const BackButton = ({ global, isEnabled, translate, unselectNote }) => {
     });
 
     return isEnabled
-        ? <a className="wpnc__back" onClick={routeBack(global, unselectNote)}>
+        ? <button className="wpnc__back" onClick={routeBack(global, unselectNote)}>
               <Gridicon icon="arrow-left" size={18} />
               {backText}
-          </a>
-        : <a className="wpnc__back disabled" disabled="disabled">
+          </button>
+        : <button className="wpnc__back disabled" disabled="disabled">
```

Comment:
> React is also smart enough to handle html5 boolean props correctly

Oh, cool – OK, since there's at least one other unrelated instance (to this PR) of this, I'll address it all together in a separate PR.

> As an aside, how do we ever get into a disabled state?

In only one situation: the Up/Down arrows on single view when at the top or bottom of the list.